tagging-video-example.md

```diff
@@ -38,7 +30,7 @@ Here is a very basic example of how you would add the title, director, date of r
 
 ## Bigger example
 
-Let's consider [the Simpsons DVDs](http://www2.foxstore.com/detail.html?item=361&u=987698787). You have a set of DVDs, each containing a few episodes of a whole season. The episodes would be spanned as follows :
+Let's consider [the Simpsons DVDs](https://en.wikipedia.org/wiki/List_of_The_Simpsons_home_video_releases). You have a set of DVDs, each containing a few episodes of a whole season. The episodes would be spanned as follows :
```

Comment:
Is it a goal to avoid using "you" here, to make things for specification-sounding?